spec/fixtures/av/la_c0652_2017_05_bag/manifest-md5.txt

```diff
@@ -0,0 +1,14 @@
+ad34e40dc8137c01c7273ade9e38e5c6 data/32101047382401.xml
```

Comment:
do these updates make the C0652 bag valid? that will be super helpful for using it as demo / dev data.

Reply:
No need I believe you. thanks for fixing them.